components/atom/upload/src/index.scss

```
@@ -8,7 +11,7 @@ $fd-atom-upload: column !default;
 $p-atom-upload: $p-l * 3 $p-l * 2 $p-l !default;
 $ai-atom-upload: center !default;
 $jc-atom-upload: center !default;
-$br-atom-upload: 0 !default;
+$br-atom-upload: $bdrs-none !default;
```

I think the variable we should use is the $bdrs-atom-upload

```suggestion
$br-atom-upload: $bdrs-atom-upload;
```